oeem-etl/utils.py

```diff
@@ -1,27 +0,0 @@
-#!/usr/bin/env python
```

Comment:
Why did this get deleted? Where will I get client_project_keys from, and S3 credentials?